src/components/@shared/FormInput/InputElement.tsx

```diff
@@ -106,17 +106,17 @@ export default function InputElement({
         />
       )
     case 'files':
-      return <FilesInput name={name} {...field} {...props} />
+      return <FilesInput name={props.name} {...field} {...props} />
```

Comment:
did you try just removing `name`? Cause it's part of `props` which we spread on component anyway